libqf/libqfqmlwidgets/src/tableview.cpp

```diff
@@ -265,8 +265,9 @@ void TableView::refreshActions()
 		curr_row = m->tableRow(toTableModelRowNo(curr_ix.row()));
 	//qfDebug() << QF_FUNC_NAME << "valid:" << r.isValid() << "dirty:" << r.isDirty();
 	if(curr_row.isDirty()) {
-		action("postRow")->setEnabled(true);
```

Comment:
all the changes must be compatible, there are some other projects using libqf, this can break old functional code